CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
 # Changelog
 
+## [v1.5.0-rc](https://github.com/googleforgames/agones/tree/v1.5.0-rc) (2020-04-07)
+
+[Full Changelog](https://github.com/googleforgames/agones/compare/v1.4.0...v1.5.0-rc)
+
+**Breaking changes:**
+
+- Upgrade to Kubernetes 1.14 [\#1329](https://github.com/googleforgames/agones/issues/1329)
```

Comment:
Hmm, we did that for 1.4. Should we manually clean up?

Reply:
We finalised the work in 1.5 - updating client go and the like.